tcms/static/js/testrun_actions.js

```diff
@@ -551,6 +551,26 @@ function changeCaseRunAssignee() {
   });
 }
 
+function updateCaseRunText() {
+  var executions = serializeCaseRunFromInputList(jQ('#id_table_cases')[0]);
```

Comment:
```suggestion
  const executions = serializeCaseRunFromInputList(jQ('#id_table_cases')[0]);
```
let's use `let` and `const` instead of `var`